docs/whats-new/2019-12.md

```diff
@@ -0,0 +1,113 @@
+---
+title: ".NET docs what's new (December 2019)"
+description: "What's new in .NET docs in December 2019."
+ms.date: 01/02/2020
+---
+
+# .NET docs what's new (December 2019)
+
+Welcome to what's new in .NET docs for December 2019. Listed below are some of the recent docs we've published.
+
+## .NET Core
+
+### New articles
+
+- [.NET Core Uninstall Tool](../core/additional-tools/uninstall-tool.md) - Documentation for .NET Core Uninstall Tool
+- [Breaking changes for migration from version 2.0 to 2.1](../core/compatibility/2.0-2.1.md) - Add struct breaking change
+- [Breaking changes for migration from .NET Framework to .NET Core](../core/compatibility/fx-core.md) - Add breaking changes doc for .NET Framework to .NET Core
+- [Tutorial: Debug a memory leak in .NET Core](../core/diagnostics/debug-memory-leak.md) - Diagnostic mem leak tutorial
+- [How to install localized IntelliSense files for .NET Core](../core/install/localized-intellisense.md) - Install localized IntelliSense
+- [How to install localized IntelliSense files for .NET Core](../core/install/localized-intellisense.md) - Install localized IntelliSense
+- [Run-time configuration options for compilation](../core/run-time-config/compilation.md) - Run-time config knobs
+- [Run-time configuration options for debugging and profiling](../core/run-time-config/debugging-profiling.md) - Run-time config knobs
+- [Run-time configuration options for globalization](../core/run-time-config/globalization.md) - Run-time config knobs
+- [Run-time configuration options for networking](../core/run-time-config/networking.md) - Run-time config knobs
+- [Run-time configuration options for threading](../core/run-time-config/threading.md) - Run-time config knobs
+- [Get started with .NET Core on Windows/Linux/macOS using the command line](../core/tutorials/cli-create-console-app.md) - Update cli tutorial for 3.1
+- [What's new in .NET Core 3.1](../core/whats-new/dotnet-core-3-1.md) - Update multiple docs for .NET Core 3.1
+
+## C# language
+
+### New articles
+
+- [Enumeration types (C# reference)](../csharp/language-reference/builtin-types/enum.md) - Moved and revised the enum article
+- [Learn about any breaking changes in the C# compiler](../csharp/whats-new/breaking-changes.md) - Add links to breaking changes doc
+
+### Updated articles
+
+- [How to join two collections (LINQ to XML) (C#)](../csharp/programming-guide/concepts/linq/how-to-join-two-collections-linq-to-xml.md) - Acrolinx 12/17/19
```

Comment:
I'd say these two are false positives because we cleaned trailing spaces since the real changes were minor